crates/aitia/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0"
 derive_more = { workspace = true }
-holochain_trace = "0.3.0-beta-dev.1"
```

Comment:
i think this become a workspace dependency

Reply:
Doesn't look like it, keystore and types moved but not trace

Reply:
oh, i forgot "could". because it's external to the repo